open-payment-platform/hertz-gateway/README.md

```diff
@@ -0,0 +1 @@
+# Gateway
```

Comment:
This file is empty, should it be filled with some descriptions? Same for the readme file in internal folder.

Reply:
It can be principle description and `generic call` link.

Reply:
done